README.md

```diff
@@ -19,6 +19,47 @@ You can use `stack build` to build `baseDAO` executable.
 
 Run `stack exec -- baseDAO --help` to see available commands.
 
+### Deploying a contract
+
+<!--
+TODO: simplify commands once we use changes in this MR:
+https://gitlab.com/morley-framework/morley/-/merge_requests/652
```

Comment:
Probably it won't be applicable after #41 anyway because there we are adding another contract to the registry.

Reply:
Yeah, fair, can remove it now.